Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/ScrollViewIsEnabled.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.IsEnabled)]
+#endif
+
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 999999, "ScrollView set to disabled will still allow scrolling", PlatformAffected.All)]
+	public class ScrollViewIsEnabled : TestNavigationPage
+	{
+		const string InitiallyEnabled = "Initially Enabled";
+		const string InitiallyNotEnabled = "Initially Not Enabled";
+		const string ToggleButton = "ToggleButton";
+		const string ScrollView = "TheScrollView";
+
+		protected override void Init()
+		{
+			var initiallyEnabled = new Button { Text = InitiallyEnabled };
+			initiallyEnabled.Clicked += (sender, args) => { Navigation.PushAsync(ScrollViewTestPage(true)); };
+
+			var initiallyNotEnabled = new Button { Text = InitiallyNotEnabled };
+			initiallyNotEnabled.Clicked += (sender, args) => { Navigation.PushAsync(ScrollViewTestPage(false)); };
+
+			var layout = new StackLayout { Children = { initiallyNotEnabled, initiallyEnabled } };
+
+			var root = new ContentPage { Content = layout };
+
+			PushAsync(root);
+		}
+
+		static ContentPage ScrollViewTestPage(bool initiallyEnabled)
+		{
+			var scrollViewContents = new StackLayout();
+			for (int n = 0; n < 100; n++)
+			{
+				scrollViewContents.Children.Add(new Label { Text = n.ToString() });
+			}
+
+			var sv = new ScrollView { Content = scrollViewContents, IsEnabled = initiallyEnabled, AutomationId = ScrollView };
```

Comment:
Nope. It wraps the line for me when the window is too narrow to show the whole thing (in Chrome, haven't checked other browsers).